tools/flecstan/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+# -*- CMake -*-
+
+project(flecstan LANGUAGES CXX)
```

Comment:
Add `VERSION 0.0.0` here.

Reply:
I've redone versioning as described in https://cmake.org/cmake-tutorial/ so that project version is specified in CMakeLists.txt and communicated through to the source code. Will upload with other changes later today.

Reply:
This tutorial is old (CMake 2.6), I was think about CMake-3 syntax (https://cmake.org/cmake/help/v3.8/command/project.html)